config.toml.in

```diff
@@ -1,46 +1,52 @@
-# An example of an optional config file for pick.
+# Pick configuration
+#
 # If no config file is available, sensible defaults will be used.
 # If you wish to use a config file, it must be located at ~/.pick/config.toml
+#
 # Variables:
-# $HOME gets replaced with the path to your home directory, e.g. $HOME/.pick -> /home/john/.pick
+# $HOME - Replaced with the path to your home directory, e.g. $HOME/.pick -> /home/john/.pick
 
-# The storage section controls the configuration of the backend.
+
+## Storage
+# The storage section controls the where the safe file is stored.
```

Comment:
`controls the where` -> `controls where`

`the safe file is stored.` -> `the safe is stored and which storage type to use.`